sentry_sdk/hub.py

```diff
@@ -322,76 +302,88 @@ def bind_client(
         top = self._stack[-1]
         self._stack[-1] = (new, top[1])
 
-    def capture_event(self, event, hint=None, scope=None, **scope_args):
+    def capture_event(self, event, hint=None, scope=None, **scope_kwargs):
         # type: (Event, Optional[Hint], Optional[Scope], Any) -> Optional[str]
         """
         Captures an event.
 
         Alias of :py:meth:`sentry_sdk.Client.capture_event`.
 
-        :param scope_args: For supported `**scope_args` see
+        :param scope_kwargs: For supported `**scope_kwargs` see
             :py:meth:`sentry_sdk.Scope.update_from_kwargs`.
         """
         client, top_scope = self._stack[-1]
-        scope = _update_scope(top_scope, scope, scope_args)
-        if client is not None:
-            is_transaction = event.get("type") == "transaction"
-            rv = client.capture_event(event, hint, scope)
-            if rv is not None and not is_transaction:
-                self._last_event_id = rv
-            return rv
-        return None
+        if client is None:
+            return None
 
-    def capture_message(self, message, level=None, scope=None, **scope_args):
+        scope_kwargs["top_scope"] = top_scope
+
+        last_event_id = client.capture_event(event, hint, scope, **scope_kwargs)
```

Comment:
Thanks for renaming all the `rv`s to something descriptive, they were making me unhappy